cmd/thyra-server/main.go

```diff
@@ -28,8 +32,8 @@ func main() {
 		}
 	}()
 
-	server.Port = 80
-	server.TLSPort = 443
+	server.Port = 8080
```

Comment:
No, I think that at the main branch level, to ensure an easier usage to enduser, we should keep 80 and 443 ports.
But we can talk about putting an environment variable to cover this usage.

Good first issue ?

Reply:
Okay, a flag can be a good idea.
By default 80 else take the flagged value